src/ast/ast.ts

```diff
@@ -51,9 +51,8 @@ import { CairoFunctionDefinition } from './cairoNodes';
 export class AST {
   // SourceUnit id -> CairoUtilFuncGen
   private cairoUtilFuncGen: Map<number, CairoUtilFuncGen> = new Map();
-  // TODO we can delete this stuff below
-  // SourceUnit id -> function signatures
-  abi: Map<number, Set<string>> = new Map();
+  // SourceUnit id -> function [signatures, returns]
+  abi: Map<number, Set<[string, string]>> = new Map();
```

Comment:
if the abi extractor is deleted there should be no writes into this; so we can delete this